### ISSUE TITLE
Help text doesn't wrap nicely with long command command help descriptions (usage)

### ISSUE THREAD

**Issue by frankcarey** [devinci-code/ahoy#15]
Instead of wrapping smartly (in a table like fashion), lines wrap to the beginning of the next line.  Not a biggie, but it can be hard to read. I've seen other commands do this where lines wrap within the column, but it probably requires knowing the terminal width. Need to look around and see if go templates support something like this.

**Comment by ocean** [comment]
Yeah I'll have a look at this as part of the new description field addition. At the moment it works but still looks a bit messy. There will almost certainly be some sort of nice Go help/console printing library, I just haven't looked any further into it yet, still using the one from `urfave/cli`, though I'm now overriding some of it with a custom output function.
